src/NServiceBus.Testing/TestingContext.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace NServiceBus.Testing
+{
+    using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using NServiceBus.Testing.ExpectedInvocations;
+
+    class TestingContext : TestableMessageHandlerContext
```

Comment:
it moves all testing framework internal additions out of the public `TestableMessageHandlerContext`.